src/test/java/org/fundacionjala/coding/abel/VigenereTest.java

```diff
@@ -0,0 +1,90 @@
+package org.fundacionjala.coding.abel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class VigenereTest {
+    private Vigenere instance;
+
+    /**
+     *
+     */
+    @Before
+    public void before() {
+        instance = new Vigenere();
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void test1() {
+        assertEquals("TXYP XMUNHCRC", instance.encode("HOLA JALASOFT", "LIMON"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "LIMON" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)